contrib/check_inst_headers.sh

```diff
@@ -28,7 +28,7 @@ do
 	fi
 
 	# try to compile a test program (from stdin) which includes hfile
-	for compile in "${CC} -x c" "${CXX} -x c++"
+	for compile in "${CC} -Werror=strict-prototypes -x c" "${CXX} -x c++"
```

Comment:
shall we pass -Werror=strict-prototypes also to CXX?

Reply:
no, in c++ strict prototypes is part of standard, if pass then compiler reports a warning